.github/workflows/__build-workflow.yaml

```diff
@@ -208,6 +208,9 @@ jobs:
       - name: Build image
         id: build
         uses: docker/build-push-action@v6.3.0
+        env:
+          DOCKER_BUILD_RECORD_UPLOAD: false
```

Comment:
The `DOCKER_BUILD_RECORD_UPLOAD: false` must be set to false as this is leading to non-redacted secrets being exposed in the oci archive summary report as pointed by the researcher